packages/env/lib/config/test/__snapshots__/config-integration.js.snap

```diff
@@ -2,6 +2,7 @@
 
 exports[`Config Integration should load local and override configuration files 1`] = `
 {
+  "afterSetup": "",
```

Comment:
Similar to the other comments, I think it'd be more clear to just not have this set, or at least have it set to null